_data/versions.yml

```diff
@@ -1346,6 +1346,68 @@ v2.1:
       version: 1.4.1
       url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=1.4.1
 
+v3.0:
+- title: v3.0.0-beta-rc1
+  note: |
+    20 October 2017
+
+    Calico v3.0.0-beta is a beta release that adds fixes and enhancements
```

Comment:
I think, with `-beta` in the name of the release, you could lose "is a beta release that"

Reply:
agreed

Reply:
Is it Calico `v3.0.0-beta` or Calico `v3.0.0-beta-rc1`? In the title we see the latter. I understand that we may have more than one beta, so I am not sure how we want to demarcate that here.

Reply:
I don't think we've settled on the correct approach. My current take is that we're releasing  a single beta version`v3.0.0-beta` and we'll append `rc[1-n]` as we cut additional "beta release candidates." If we need additional beta releases, then `v3.0.1-beta` is an option.

This looks more or less like what we do today, except we're adding `-beta` to the release name.

Another dimension, if we need it, is to release incremental beta releases, e.g. `beta1`, `beta2`:
 - `v3.0.0-beta1-rc1`, `v3.0.0-beta1-rc2`, and release `v3.0.0-beta1` 
 - `v3.0.0-beta2-rc1`, and release `v3.0.0-beta2` 

cc @caseydavenport WDYT?